Addition: Reason for not importing stats with `from sympy import *`

czgdp1807 opened the pull request:
<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests .-->
N/A

#### Brief description of what is fixed or changed
stats was not imported with `from sympy import *` earlier.
This commit fixes the above issue.

#### Other comments
I don't know the reason for not including `from .stats import *` in `__init__.py`. However,
when I used `from sympy import *`, it didn't import the `stats` module and hence this compelled
me to write a new line `from sympy.stats import  * `. I made this PR just for the ease of the user who
may face an error while working with stats module.
If I am wrong somewhere, then please correct me. If this change can cause performance issues then an explanation will be helpful. :-)
Note - After some discussion in the below comments with @ethankward, the change has been commented out and reason for doing the same has been mentioned.

#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
* stats
  * reason for not importing stats with `from sympy import *` has been included
<!-- END RELEASE NOTES -->

gdp1807 committed:
stats is now imported with `from sympy import *`

stats was not imported with `from sympy import *` earlier.
This commit fixes the above issue.

czgdp1807 commented:
Yeah, some tests failed and made it clear. :-)

ethankward commented:
I would probably raise this as an issue before doing a pull request as many major changes will have to be made and discussed.

czgdp1807 commented:
I don't think we should change the stats module for import issues. Any change in interface will create further issues for current users. That's why I didn't raise an issue for this because this is not a problem but only an addition. Even `combinatorics` module isn't imported directly, with, `from sympy import *`, however, the reason for the same is mentioned in the comments. 
So, according to my point of view, adding a reason would be better for not importing `stats` rather than resolving name conflicts.

czgdp1807 committed:
Added the reason for not importing stats

Importing `stats` module with `from sympy import *` causes conflicts, and hence the line has been commented out along with the addition of the reason.
